Story/Voltaire/11BattledoomStory.cs

```diff
@@ -0,0 +1,80 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class Battledoom
```

Comment:
why this is in a seperate file? not in the core

Reply:
Because his structure, see the file